src/cli/serve/serve.js

```diff
@@ -80,7 +77,9 @@ module.exports = function (program) {
   .option(
     '-c, --config <path>',
     'Path to the config file, can be changed with the CONFIG_PATH environment variable as well',
```

Comment:
Want to add a note to this option description to indicate that it can be called multiple times in order to read from more than one config?